examples/complete/config/service-control-policies/iam-policies.yaml

```diff
@@ -0,0 +1,24 @@
+- sid: "DenyCreatingIAMUsers"
+  effect: "Deny"
+  actions:
+    - "iam:CreateUser"
+    - "iam:CreateAccessKey"
+  resources:
+    - "*"
+
+
+- sid: "ProtectIAMRoles"
```

Comment:
Same for these... let's remove. Create simple examples like "test1.yaml" and "test2.yaml"